services/src/main/scala/cromwell/pipeline/service/ProjectConfigurationService.scala

```diff
@@ -31,21 +25,21 @@ object ProjectConfigurationService {
     new ProjectConfigurationService {
 
       def addConfiguration(projectConfiguration: ProjectConfiguration, userId: UserId): Future[Unit] = {
-        def toTuple(config: ProjectFileConfiguration): (Path, Map[String, TypedValue]) =
-          config.path -> config.inputs.map(fileParameter => fileParameter.name -> fileParameter.typedValue).toMap
+        def toTuple(wdlParams: WdlParams): (Path, Map[String, TypedValue]) =
+          wdlParams.path -> wdlParams.inputs.map(fileParameter => fileParameter.name -> fileParameter.typedValue).toMap
 
-        def toProjectFileConfiguration(path: Path, inputs: Map[String, TypedValue]): ProjectFileConfiguration =
-          ProjectFileConfiguration(path, inputs.map((FileParameter.apply _).tupled).toList)
+        def toWdlParams(path: Path, inputs: Map[String, TypedValue]): WdlParams =
+          WdlParams(path, inputs.map((FileParameter.apply _).tupled).toList)
 
         getLastByProjectId(projectConfiguration.projectId, userId).flatMap {
           case Some(config) =>
-            val oldFileConfigsMap = config.projectFileConfigurations.map(toTuple).toMap
-            val newFileConfigsMap = projectConfiguration.projectFileConfigurations.map(toTuple).toMap
+            val oldWdlParamsMap = List(config.wdlParams).map(toTuple).toMap
+            val newWdlParamsMap = List(projectConfiguration.wdlParams).map(toTuple).toMap
 
-            val updatedFileConfigsMap = oldFileConfigsMap ++ newFileConfigsMap
-            val updatedFileConfigs = updatedFileConfigsMap.map((toProjectFileConfiguration _).tupled).toList
+            val updatedWdlParamsMap = oldWdlParamsMap ++ newWdlParamsMap
+            val updatedWdlParams = updatedWdlParamsMap.map((toWdlParams _).tupled).head
```

Comment:
Please avoid using unsafe operations like `head` or `get` in your code. Usually it shows that you are doing something wrong

Reply:
Removed this piece